samples/webapps/jaxrs_basic/pom.xml

```diff
@@ -39,13 +39,15 @@
             </plugin>
             <plugin>
                 <artifactId>maven-war-plugin</artifactId>
+                <version>3.1.0</version>
                 <configuration>
```

Comment:
Shouldn't we inherit these versions from parent pom?

Reply:
This plugin is not used in any other pom, so does it make sense to add it to parent pom?

Reply:
Just add version in the root pom and refer in here.

Reply:
Fixed with 
15df4ec